snippets/paper-elements.cson

```diff
@@ -9,3 +9,21 @@
       <div main>$2</div>
     </paper-drawer-panel>
     """
+  "paper toolbar":
+    "prefix": "paper-toolbar"
+    "body": """
+    <paper-toolbar>
+      <paper-icon-button icon="$1" on-tap="menuAction"></paper-icon-button>
+      <div title>$2</div>
+      <paper-icon-button icon="$3" on-tap="moreAction"></paper-icon-button>
+    </paper-toolbar>
+    """
+  "paper tabs":
+    "prefix": "paper-tabs"
+    "body": """
+    <paper-tabs selected="0">
+      <paper-tab>$1</paper-tab>
```

Comment:
Should we put some default values in these tabs so they are always visible? For example, fast prototyping. Something like

    <paper-tab>${1:Tab 1}</paper-tab>